### PR TITLE
Create fr.json

### PR DESCRIPTION
**Pull request by SkyzohKey**
This is the French translation, 100% complete.

### CODE REVIEW

File: tl/fr.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+    "multi": {
+        "_lang_native": "French",
```
Comment (stal888) [inline tl/fr.json]: should be "Français"

**Comment by SkyzohKey** [comment]
This is ready to be merged. :)

**Comment by Zer0-One** [comment]
@SkyzohKey not anymore it isn't, lol

**Comment by SkyzohKey** [comment]
@Zer0-One Hmm, i'll have to redo this. Working on Ricin's groupchats & bugs right now. I'll do it after that :)

**Pull request closed without merging**